cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt

```diff
@@ -0,0 +1,666 @@
+      'Dimensions.member.1.Value': hostname,
+      'StartTime': start_date,
+      'EndTime': end_date,
+      'Period': "2592000",
+      'Statistics.member.1': 'Maximum',
+      'Statistics.member.2': 'Average',
+      'Statistics.member.3': 'Minimum'
+    }
+  }
+EOS
+end
+
+script "js_merged_metrics", type: "javascript" do
+  parameters "ds_cloudwatch_cpu_maximum", "ds_cloudwatch_disk_usage", "ds_cloudwatch_mem_usage"
+  result "merged_results"
+code <<-EOS
+  var merged_results = []
+  for ( i = 0; i < ds_cloudwatch_cpu_maximum.length; i++){
+    var instance_id = ds_cloudwatch_cpu_maximum[i].instanceId
+    var disk_usage_record = _.find(ds_cloudwatch_disk_usage, function(record) { return record.instanceId == instance_id; })
+    if (disk_usage_record != null && disk_usage_record !== undefined) {
+      var disk_maximum = parseFloat(disk_usage_record.disk_maximum).toFixed(2)
+      var disk_average = parseFloat(disk_usage_record.disk_average).toFixed(2)
+      var disk_minimum = parseFloat(disk_usage_record.disk_minimum).toFixed(2)
+    } else {
+      var disk_maximum = "N/A"
+      var disk_average = "N/A"
+      var disk_minimum = "N/A"
+    }
+    var mem_usage_record = _.find(ds_cloudwatch_mem_usage, function(record) { return record.instanceId == instance_id; })
+    if (mem_usage_record != null && mem_usage_record !== undefined) {
+      var mem_maximum = parseFloat(mem_usage_record.mem_maximum).toFixed(2)
+      var mem_average = parseFloat(mem_usage_record.mem_average).toFixed(2)
+      var mem_minimum = parseFloat(mem_usage_record.mem_minimum).toFixed(2)
+    } else {
+      var mem_maximum = "N/A"
+      var mem_average = "N/A"
+      var mem_minimum = "N/A"
+    }
+    merged_results.push(
+      {
+        "region": ds_cloudwatch_cpu_maximum[i].region,
+        "instanceId": ds_cloudwatch_cpu_maximum[i].instanceId,
+        "privateDnsName": ds_cloudwatch_cpu_maximum[i].privateDnsName,
+        "hostname": ds_cloudwatch_cpu_maximum[i].privateDnsName.split('.')[0],
+        "tag_set": ds_cloudwatch_cpu_maximum[i].tag_set,
+        "cpu_minimum": parseFloat(ds_cloudwatch_cpu_maximum[i].cpu_minimum).toFixed(2),
+        "cpu_maximum": parseFloat(ds_cloudwatch_cpu_maximum[i].cpu_maximum).toFixed(2),
+        "cpu_average": parseFloat(ds_cloudwatch_cpu_maximum[i].cpu_average).toFixed(2),
+        "disk_maximum": disk_maximum,
+        "disk_average": disk_average,
+        "disk_minimum": disk_minimum,
+        "mem_maximum": mem_maximum,
+        "mem_minimum": mem_minimum,
+        "mem_average": mem_average
+      }
+    )
+  }
+EOS
+end
+
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_utilization" do
+  validate_each $ds_merged_metrics do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing instance data"
```

Comment:
The summary should include more relevant info about the specific policy.
`summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing AWS instance CloudWatch utilization data"` ??